### ISSUE TITLE
Suggestion: When row isn't defined in file, fallback to default instead of null.

### ISSUE THREAD

**Issue by alexanderhorner** [ryangjchandler/orbit#107]
Right now, the behavior for an undefined row is to just insert null, wich gives an error if the field isn't nullable. I think it would be better to fallback to default if it is undefined, just how the `Model::create()` method would do.

**Comment by ryangjchandler** [comment]
@alexanderhorner Where are you defining the default value, in the schema?

**Comment by alexanderhorner** [comment]
Yea

```php
class Article extends Model
{
    use Orbital;

    protected $fillable = ['title', 'link', 'category', 'author', 'published_on', 'thumbnail', 'thumbnail_slide'];

    public $timestamps = false;

    public static function schema(Blueprint $table)
    {
        $table->string('title');
        $table->string('link');
        $table->string('category')->default("Allgemein");
        $table->string('author')->nullable()->default(null);
        $table->timestamp('published_on')->nullable()->default(null); // 'YYYY-MM-DD HH:MI:SS'
        $table->string('thumbnail')->default('missing_thumbnail');
        $table->string('thumbnail_slide')->default('missing_thumbnail_slide');
    }
```

**Comment by ryangjchandler** [comment]
This _should_ be happening already.

**Comment by ryangjchandler** [comment]
There's a test for this, `DefaultDatabaseValuesTest` and it passes as expected.

**Comment by alexanderhorner** [comment]
Are you sure? If the test is creating the files through Orbit, the default values are inserted in the file (that's what happens when you call Model::create(). But if you have a file with the row simply missing, even if a default value for that row is defined, it will through the usual error `'SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: articles.category` (in this case category is the missing line in the file, with the default value.

**Comment by ryangjchandler** [comment]
@alexanderhorner Right, I see now. Easier if you provide the exception in the original issue description. 

I'll take a look at this.

**Comment by alexanderhorner** [comment]
Apologies, I thought you new about that since its such a common error for me. Everytime a row isn't defined in a file this error gets thrown. Would be cool if it'd only affect the file with the missing row, not the whole Model though.

**Comment by ryangjchandler** [comment]
@alexanderhorner Now available in v1.1.3 :)
